rust/cymbal/src/pipeline/exception/stack_processing.rs

```diff
@@ -98,7 +98,15 @@ pub async fn do_stack_processing(
                 .map_err(|e| (index, e))?
         }
 
-        let proposed = generate_fingerprint(&props.exception_list);
+        let team_id = events[index]
+            .as_ref()
+            .expect("no events have been dropped since indexed-property gathering")
+            .team_id;
+
+        let proposed = resolve_fingerprint(context.clone(), team_id, &props)
```

Comment:
If this is returning the fingerprint based on a matching rule rather than running the normal generation will the "proposed_fingerprint" property on the exception event be that of the rule? Should we be setting it to the default algorithm output instead?

Reply:
Fair, not against that idea.